src/main/java/com/internet/shop/controller/IndexController.java

```diff
@@ -0,0 +1,16 @@
+package com.internet.shop.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet({"/","/home"})
```

Comment:
do we need to have two paths?

Reply:
I think. this is test version. And i try take two paths to controller.  But will change that